app-config.yaml

```diff
@@ -271,6 +271,10 @@ catalog:
     # Example component for TechDocs
     - type: file
       target: ../../plugins/techdocs-backend/examples/documented-component/catalog-info.yaml
+    - type: url
+      target: https://github.com/acierto/software-templates/blob/main/scaffolder-templates/docs-long-running-template/template.yaml
+      rules:
+        - allow: [Template]
```

Comment:
Let's not add this to the actual app config for everyone

Reply:
Oh sorry, that was unintended commit